backend/model/model.go

```diff
@@ -1413,6 +1413,7 @@ type Graph struct {
 	GroupByKey        *string
 	BucketByKey       *string
 	BucketCount       *int
+	BucketInterval    *int
```

Comment:
how does this get used?

Reply:
wasn't being used in original PR but fixed now - this is just for saving / loading the graph's bucket interval setting